isisd/isis_cli.c

```diff
@@ -452,6 +452,29 @@ void cli_show_isis_overload_on_startup(struct vty *vty,
 		yang_dnode_get_string(dnode, NULL));
 }
 
+/*
+ * XPath: /frr-isisd:isis/instance/advertise-high-metrics
+ */
+DEFPY_YANG(advertise_high_metrics, advertise_high_metrics_cmd,
+	   "[no] advertise-high-metrics",
+	   "Restore configured metric value on all interfaces\n"
+	   "Advertise high metric value on all interfaces\n")
+{
+	nb_cli_enqueue_change(vty, "./advertise-high-metrics", NB_OP_MODIFY,
+			      no ? "false" : "true");
+
+	return nb_cli_apply_changes(vty, NULL);
+}
+
+void cli_show_advertise_high_metrics(struct vty *vty,
+				     const struct lyd_node *dnode,
+				     bool show_defaults)
+{
+	if (!yang_dnode_get_bool(dnode, NULL))
+		vty_out(vty, " no");
```

Comment:
What needs to be changed here?

Reply:
When you do "show run":
if advertise-high-metrics is on, you display "advertise-high-metrics"
but if not, you don't display "no advertise-high-metrics" because it is a default value.
So you only display "no advertise-high-metrics" when off and `show_defaults `is true

Reply:
Updated